classes/UI/PagesComponent.php

```diff
@@ -128,12 +129,13 @@ public function ShowHeader($id_string, OptionsInterface $options, Application $a
 
 		if ($options->Get('title') !== '')
 		{
-			$args['custom_title.body'] = $options->Get('title');
-			$args['custom_title.visible'] = 1;
+			$args['custom_title.body']     = cla_htmlsafe(ClaText::ProcessAvailableLocalisation((string) $options->Get('title')));
```

Comment:
You don't need cla_htmlsafe is you use ".body" templater variable. ;)

(I just wonder, why did Github send me a notification about this PR?)

Reply:
Nicely spotted! This change was made for very many modules, and most others used `.body_html`, so this one must have slipped by.

I'm not sure why GitHub would have sent you an email though, perhaps you were still watching the repository? 😄